core/domain/src/main/java/com/projectlab/core/domain/model/User.kt

```diff
@@ -0,0 +1,11 @@
+package com.projectlab.core.domain.model
+
+data class User(
+    var id: String = "",
```

Comment:
why is it "id" a var instead of val? I don't think it should change once created